src/flint/fmpz.jl

```diff
@@ -1373,6 +1391,7 @@ function next_prime(x::fmpz, proved::Bool = true)
 end
 
 function next_prime(x::UInt, proved::Bool = true)
+   x >= 0xffffffffffffffc5 && error("No larger single-limb prime exists")
```

Comment:
```suggestion
```
No. This is 64-bit specific, and it is already done correctly. See test code.

Reply:
True it is 64 bit specific, but the flint exception prints an error in our tests which shows up in the test output.

Reply:
This is what we want, right?

Reply:
Hopefully correct this time. Can't check as Nemo is using the latest AA which is broken without the corresponding Nemo PR's merged.

Reply:
Why would we want an error message to print while our tests are running?

Reply:
If you don't want to see the error messages from flint printed to stdout, then the exception system in flint needs an overhaul.

Reply:
Yes it does.

Reply:
You should see the singular.jl test code: we print all of the error messages and it is fine.